assets/tsconfig.json

```diff
@@ -27,7 +27,7 @@
 
     /* Strict Type-Checking Options */
     "strict": true /* Enable all strict type-checking options. */,
-    "noImplicitAny": true /* Raise error on expressions and declarations with an implied 'any' type. */,
+    "noImplicitAny": false /* Raise error on expressions and declarations with an implied 'any' type. */,
```

Comment:
```suggestion
    "noImplicitAny": false /* Do not raise error on expressions and declarations with an implied 'any' type. */,
```

Reply:
In my mind the comments are "static" documentation of what the options do when enabled, so under that logic I'd keep it the same, if that makes sense. (Actually I'd prefer not having the comments at all, since they're noisy, duplicative of the `tsconfig` documentation, and can easily get out of sync with it, but thought to save that idea for another day.)

Reply:
Yep, makes sense.